### DISCUSSION TITLE
Help setting up code server for a newbie

### DISCUSSION THREAD

**Discussion by Frityet** [coder/code-server#2561] [Q&A]
Hey y'all, I recently dual booted Ubuntu Server on my pc, and I want to run code server on it. I am using the lets encrypt-caddy method, but I cannot get the server to be accessible from my local network, let alone the internet. I do have a domain, and I followed everything to how it was told (i hope)

**Comment by Frityet** [comment]
I have portforwarded port 8080 btw

**Reply by code-asher** [reply]
If you're using Caddy you won't need to forward 8080 but you will want to forward 80 and 443. The browser will hit 80/443 then Caddy will forward it locally to 8080.

**Reply by code-asher** [reply]
You might not be able to access it locally because of certificate issues; if forwarding 80 & 443 doesn't work then could you post what error you get?
